tests/Avalonia.Visuals.UnitTests/Rendering/DeferredRendererTests.cs

```diff
@@ -370,7 +370,7 @@ private void IgnoreFirstFrame(Mock<IRenderLoop> loop, Mock<ISceneBuilder> sceneB
 
         private void RunFrame(Mock<IRenderLoop> loop)
         {
-            loop.Raise(x => x.Tick += null, EventArgs.Empty);
+            //loop.Raise(x => x.Tick += null, EventArgs.Empty);
```

Comment:
is this meant to be left commented out?

Reply:
Nope! Sorry, this PR is still WIP, will fix that!